components/CategorySelection/index.js

```diff
@@ -30,13 +22,14 @@ const CategoriesSelection = ({ url, title }) => {
         </p>
       </div>
       <div className="container mx-auto">
-        <div className="flex flex-wrap items-center justify-center">
-          {arrayData.slice(0, 3).map((item) => (
+        <div className="flex flex-wrap items-center justify-center font-bold">
+          {data.data.map((item) => (
```

Comment:
You could destructure `item` here Negin to avoid repeating `item.` on lines 28-32